docs/QuickStart-Rich-Styling.md

```diff
@@ -64,7 +64,7 @@ class MyEditor extends React.Component {
     return (
       <Editor
         editorState={this.state.editorState}
-        handleKeyCommand={this.handleKeyCommand}
+        handleKeyCommand={this.handleKeyCommand.bind(this)}
```

Comment:
I'd prefer if this were instead bound in the constructor, in line with current ES6 React patterns.